homeassistant/components/media_player/sonos.py

```diff
@@ -138,9 +138,14 @@ def update(self):
         """Retrieve latest state."""
         self._name = self._player.get_speaker_info()['zone_name'].replace(
             ' (R)', '').replace(' (L)', '')
-        self._status = self._player.get_current_transport_info().get(
-            'current_transport_state')
-        self._trackinfo = self._player.get_current_track_info()
+
+        if self._is_reachable:
+            self._status = self._player.get_current_transport_info().get(
```

Comment:
It won't since we catch the exception without throwing it again 

But I like the status idea! :)